app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_import.scala.html

```diff
@@ -62,18 +74,25 @@ <h2 class="govuk-heading-l">@messages("inspection_needed_import.what_next.h2")</
                         if (optLocations.isEmpty){
                             List(Html(
                                 s"""<h3 class="govuk-heading-m">${messages("inspection_needed_import." + inspectionType.description.toLowerCase + ".heading")}</h3>
-                           |<p class="govuk-body">${messages("inspection_needed_import_with_other.p1")}</p><hr aria-hidden="true" /><br/> """.stripMargin))
+                           |<p class="govuk-body">${messages("inspection_needed_import_with_other.p1")}</p>""".stripMargin))
                         } else {
-                            renderInspectionLocations(inspectionType, optLocations) :+ Html("<hr aria-hidden=\"true\" /><br />")
+                            renderInspectionLocations(inspectionType, optLocations)
                         }
                 }
         } else {
                 Html(
                     s"""<h3 class="govuk-heading-m">${messages("inspection_needed_import.inspection.heading")}</h3>
-                       |<p class="govuk-body">${messages("inspection_needed_import_with_other.p1")}</p> <hr aria-hidden="true" /><br/>""".stripMargin)
+                       |<p class="govuk-body">${messages("inspection_needed_import_with_other.p1")}</p> """.stripMargin)
         }
     }
 
+    @govukButton(
```

Comment:
You haven't removed the "Check another GMR" hyperlink, have you? ie. you've added in the button, but the hyperlink is still there, I think.

Reply:
removed